src/commands/create_drive.ts

```diff
@@ -12,17 +13,25 @@ import { Wallet, JWKWallet, PrivateDriveKeyData } from 'ardrive-core-js';
 
 new CLICommand({
 	name: 'create-drive',
-	parameters: [...DriveCreationPrivacyParameters, DriveNameParameter, BoostParameter, DryRunParameter],
+	parameters: [
+		...DriveCreationPrivacyParameters,
```

Comment:
Order these in a sequence that makes for the most friendly help output when you do `ardrive create-drive --help`. Refer to the ordering for `upload-file` on the multifile upload branch for insp.